articles/models.py

```diff
@@ -202,7 +202,7 @@ def __str__(self):
 class FeatureStyleFields(models.Model):
     feature_style = models.ForeignKey(
         FeatureStyle,
-        default=1,
+        default=2,
```

Comment:
This number corresponds to the id of the FeatureStyle model that we want to be the new default.  It is the second FeatureStyle created in the 0022_create_feature_styles.py migration.